pkg/api/annotations/pod.go

```diff
@@ -27,6 +27,14 @@ const PodLaunchType = "pod.elotl.co/launch-type"
 // override specified resource requests and limits.
 const PodInstanceType = "pod.elotl.co/instance-type"
 
+// PodImageOverlayRootfs is an annotation users add to their
+// kubernetes pods to inform kip whether the itzo cell will use
+// -mount or -extractto argument when itzo calls tosi to pull rootfs
+// images. This annotation will default to -mount if its value is empty
+// or truthy, if the value is falsy it will use the -extractto option.
+// Allowed values: 1, t, T, TRUE, true, True, 0, f, F, FALSE, false, False
+const PodImageOverlayRootfs = "pod.elotl.co/image-overlay-rootfs"
```

Comment:
@ldx: do we want to test this out and make sure it makes a difference before publishing docs on how to use it and making the annotation public, to last for all time?  If we do want this to be available to users, docs/annotations.md should be updated with the comment above.

Reply:
@ldx just so we can get this pr out of the way. Should I be adding this annotation to the `docs/annotations.md`? This PR is in reference to allowing the use of direct extraction of images vs using an overlayfs.

Reply:
Yeah, let's add it to annotations.md :+1: